src/app/shared/services/app-events/app-events.service.ts

```diff
@@ -79,6 +93,7 @@ export class AppEventService extends AsyncServiceBase {
     // current app day calculated by finding length of array of subset of all unique app open event dates
     const app_day = [...new Set(appOpenEvents.map((e) => e._created.substring(0, 10)))].length;
     const first_app_launch = this.appEventsById.app_launch?.[0]?._created || generateTimestamp();
+    this.localStorageService.setString(this.appFields.APP_FIRST_LAUNCH, first_app_launch);
```

Comment:
nit(blocking) - something doesn't quite feel right here, we're setting the first_launch field any time event summaries are calculated, which is not only on init but also any time there is any other recorded app event. Really this should only ever happen once, on initial load and only if not already populated. 

I'd probably suggest moving the logic to the main `app.component` for now where we set other hardcoded fields, and just do it somewhat deferred after everything else has been initialised and only if there is not an existing value